### ISSUE TITLE
genesyscloud_group_roles - cannot assign 2 divisions.

### ISSUE THREAD

**Issue by PiotrPardo** [MyPureCloud/terraform-provider-genesyscloud#707]
Hello,

After update to 1.24(same for 1.25 and 1.25.1) there is a problem with assignment of more than 1 division to role in group role.

example code:

```
resource "genesyscloud_group_roles" "TestGroupRoles" {
  group_id = genesyscloud_group.TestGroup.id
  roles {
    role_id = genesyscloud_auth_role.SupervisorRole.id
    division_ids = [genesyscloud_auth_division.DivisionOne.id,genesyscloud_auth_division.DivisionTwo.id]
  }
  roles {
    role_id      = genesyscloud_auth_role.AgentRole.id
    division_ids = [genesyscloud_auth_division.DivisionOne.id]
  }
  roles {
    role_id = genesyscloud_auth_role.AgentDetails.id
    division_ids = [genesyscloud_auth_division.DivisionOne.id]
  }
}
```

Error after apply

```
Error:mismatch on attribute roles.1.division_ids.1234567890:
expected value: <nil>
actual value:   XXX-XXX-XXX-XXX-XXX

  with genesyscloud_group_roles.TestGroupRoles,
  on genesyscloud_group_roles.tf line 2013, in resource "genesyscloud_group_roles" "TestGroupRoles":
2013: resource "genesyscloud_group_roles" "TestGroupRoles" 
```

**Comment by charliecon** [comment]
Hi @PiotrPardo 

I was able to recreate this bug you're seeing. We will open a ticket for this and get a fix out as soon as we can.

Thanks for reporting!

-Charlie

**Comment by charliecon** [comment]
(Tracking with DEVTOOLING-239)

**Comment by charliecon** [comment]
Fix released on v1.26.0
